extensions/amp-twitter/1.0/component.js

```diff
@@ -43,8 +43,9 @@ function TwitterWithRef({requestResize, title, ...rest}, ref) {
         const height = data['height'];
         if (requestResize) {
           requestResize(height);
+          setHeightStyle(NO_HEIGHT_STYLE);
         } else {
-          setHeightStyle(height);
+          setHeightStyle({height});
```

Comment:
Actually, can we go in the other direction? Making the `height` a string allows the naive dirty checking in `useState` to skip a diff if the height is the same, but it won't be able to do that if we have an object.

Reply:
Done.